src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -168,6 +168,20 @@ def is_bicep_file(file_path):
     return file_path.lower().endswith(".bicep")
 
 
+def is_bicepparam_file(file_path):
+    return file_path.lower().endswith(".bicepparam")
+
+
+def is_bicepparam_file_provided(parameters):
```

Comment:
Personally, I think it would be better to move the helper method to custom.py, since `parameters` is a command argument.